### ISSUE TITLE
BAccessibility: Pressing Shift+Tab key in Ctrl+F moves to terminal area

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#181060]
CC @meganrogge

1. Load NVDA.

1. Open a new editor.

1. Make sure your tabFocusMode is on.

1. Press Ctrl+F

1. Press Tab key and hit Shift Tab key.

Notes: Your focus moves to terminal area.

## System Info

Version: 1.78.0-insider (system setup)
Commit: d6290ef6582acecbf3bca1558111f2ee64375801
Date: 2023-04-27T05:20:48.295Z
Electron: 22.4.8
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
When I follow those steps (with a normal editor, not a terminal in the editor area), the input in the find widget is selected - not the terminal. 

I am going to try to reproduce with a terminal in the editor area

**Comment by jooyoungseo** [comment]
@meganrogge -- In the Ctrl+F view, press Tab and Shift+Tab and press Tab again. My focus keeps moving to terminal.

**Comment by meganrogge** [comment]
I cannot reproduce. @isidorn can you try to?

**Comment by isidorn** [comment]
Cannot reproduce. @jooyoungseo maybe it is one of the extensions you have installed? Can you reproduce with a new profile or starting `code-insiders --user-data-dir SOME_PATH_ON_YOUR_DISK`

**Comment by jooyoungseo** [comment]
After following @isidorn's suggestion, I could not reproduce this issue. Maybe, it has to do with one of my extensions. I will reopen this once I figure this out.
